lib/galaxy/tool_shed/galaxy_install/tools/tool_panel_manager.py

```diff
@@ -109,24 +112,24 @@ def generate_tool_elem(self, tool_shed, repository_name, changeset_revision, own
                            tool, tool_section):
         """Create and return an ElementTree tool Element."""
         if tool_section is not None:
-            tool_elem = XmlET.SubElement(tool_section, 'tool')
+            tool_elem = etree.SubElement(tool_section, 'tool')
         else:
-            tool_elem = XmlET.Element('tool')
+            tool_elem = etree.Element('tool')
```

Comment:
Need to get `Element` first with `Element = getattr(etree, '_Element', etree.Element)` as you did elsewhere?